<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DonateDialog</class>
 <widget class="QDialog" name="DonateDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>223</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="labelThanks">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>85</y>
     <width>411</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Thank you for using Keibo-MoneyTracker. If you find this app useful please consider donating. </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="CustomPushButton" name="pushButtonDonate">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>165</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Donate</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsViewIcon">
   <property name="geometry">
    <rect>
     <x>215</x>
     <y>10</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0, 0);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomPushButton</class>
   <extends>QPushButton</extends>
   <header location="global">custompushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
